### DISCUSSION TITLE
Modules file path in tic-80

### DISCUSSION THREAD

**Discussion by spooneystone** [nesbox/TIC-80#2078] [General]
Hi all,

I am quite new to coding lua, so looking to get advice on this problem before i report it in issues.

When implementing a module i get this error 
![tic_80_](https://user-images.githubusercontent.com/17956156/211657507-be66b861-7174-4919-9b28-e500eddd141a.PNG)

However, i have the lua files in the same directory, see here

![Files_location](https://user-images.githubusercontent.com/17956156/211657652-248a4264-c7bf-4a55-b029-3775bf0ea57f.PNG)

Why is it rooted to that directory?
and can i change it?

If i place the test.lua in that directory, then it works as it should.

Any help or idea would be great.

**Comment by joshgoebel** [comment]
Multiple files are not supported, your source must all live in a single large file.  Alternatively you may want to look into bundlers: https://github.com/nesbox/TIC-80/wiki/tools#bundling

**Reply by spooneystone** [reply]
Ok, thanks for clearing that up for me, but it works if i put the file in the directory it shows the error in. 
just want to mention that for people to take note of.

**Reply by joshgoebel** [reply]
Sounds like a bug.

**Reply by nesbox** [reply]
It works for only local scripts only and it won't work if you try to export your game.

**Reply by joshgoebel** [reply]
This seems very strange to me. Should we remove/hide that lua functionality in a future release?

**Reply by nesbox** [reply]
Yes, we have to disable lua imports somehow...

**Reply by joshgoebel** [reply]
Is import a keyword of the language vs a function we can just override?

**Reply by nesbox** [reply]
`loadfile()` and `dofile()` are already disabled here https://github.com/nesbox/TIC-80/blob/main/src/api/lua.c#L1488
also, you can use `require ...` to import module which is not disabled
